internal/stacks/tfstackdata1/convert.go

```diff
@@ -99,6 +99,12 @@ func DynamicValueToTFStackData1(val cty.Value, ty cty.Type) (*DynamicValue, erro
 	return ret, nil
 }
 
+func DynamicValueFromTFStackData1(protoVal *DynamicValue, ty cty.Type) (cty.Value, error) {
+	// FIXME: Apply sensitive marks to everything in protoVal.SensitivePath
```

Comment:
Is it reasonable to do this marking now, or at least as a quick followup? I'd rather not introduce a new case where we can accidentally reveal sensitive values if at all possible.

Reply:
Ahh, yes... for some reason I'd convinced myself you'd solved that already in https://github.com/hashicorp/terraform/pull/34414, but I think I was getting confused with a similar thing you did in some other place.

I have to assume we already have some code somewhere dealing with something similar to this, so I'll see if I can track it down and move it into a more shareable place rather than writing it over again here.

Reply:
I'm going to land this PR with this as-is, but I'll start working on another PR to fill this gap immediately after.

(Just because there's already enough going on in this PR as it is, and I want to merge it to allow independent PRs to follow without coming in to conflict.)

Reply:
Resolved in https://github.com/hashicorp/terraform/pull/34505